ispdb/jet.ne.jp.xml

```diff
@@ -5,25 +5,26 @@
     <displayName>JETINTERNET</displayName>
     <displayShortName>JET</displayShortName>
     <incomingServer type="pop3">
-      <hostname>pop.jet.ne.jp</hostname>
-      <port>110</port>
-      <socketType>plain</socketType>
+      <hostname>pop02.jet.ne.jp</hostname>
+      <port>995</port>
+      <socketType>SSL</socketType>
       <username>%EMAILLOCALPART%</username>
-      <authentication>password-encrypted</authentication>
+      <authentication>password-cleartext</authentication>
```

Comment:
This is a downgrade. Did you verify that the encrypted password really is no longer possible?

(I understand that the connection is SSL-encrypted. But plaintext passwords are vulnerable to attacks on either end, either on client using imported SSL CAs, or via server hacks, see haveibeenpwned.com )

Reply:
The server only advertises `SASL PLAIN LOGIN` via POP3, `AUTH=PLAIN AUTH=LOGIN` via IMAP, and `AUTH PLAIN LOGIN` via SMTP.